src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -23,8 +23,23 @@ public class FullNodeController : FeatureController
         /// <summary>Instance logger.</summary>
         private readonly ILogger logger;
 
+        private readonly IPooledTransaction pooledTransaction;
```

Comment:
Apro is gonna want comments 😉

Reply:
Fixed: added comments for those components that are commented in the codebase. I didn't touch those that are never commented because this is probably something for another PR.